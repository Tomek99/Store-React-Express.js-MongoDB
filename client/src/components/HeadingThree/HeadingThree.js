import React from "react";
import PropTypes from "prop-types";
import styles from "./HeadingThree.module.scss";

function HeadingThree({ title }) {
  return <h3 className={styles.HeadingThree}>{title}</h3>;
}

HeadingThree.propTypes = {
  title: PropTypes.string,
};
export default HeadingThree;
