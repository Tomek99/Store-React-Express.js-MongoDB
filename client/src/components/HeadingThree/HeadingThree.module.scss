.HeadingThree {
  font-size: 2rem;
  font-weight: 400;
  color: var(--text-color);
}
