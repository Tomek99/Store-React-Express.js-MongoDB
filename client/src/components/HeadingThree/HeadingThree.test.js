import { render } from "@testing-library/react";
import HeadingThree from "./HeadingThree";

test("should render HeadingThree component", () => {
  const { container } = render(<HeadingThree title="Test" />);
  expect(container).toMatchSnapshot();
});
