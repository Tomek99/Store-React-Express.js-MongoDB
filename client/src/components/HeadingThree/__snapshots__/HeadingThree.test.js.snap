// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render HeadingThree component 1`] = `
<div>
  <h3
    class="HeadingThree"
  >
    Test
  </h3>
</div>
`;
